php_phongo.c

```diff
@@ -451,10 +451,63 @@ mongoc_bulk_operation_t *phongo_bulkwrite_init(zend_bool ordered) { /* {{{ */
 #define PHONGO_WRITECONCERN_ALLOWED   0x01
 #define PHONGO_READPREFERENCE_ALLOWED 0x02
 
-static int process_read_preference(zval *option, zval **zreadPreference TSRMLS_DC)
+static int process_read_concern(zval *option, bson_t *mongoc_opts TSRMLS_DC)
```

Comment:
`php_phongo_command_opts_append_document()` returns a `bool`, so this should, too.

Reply:
Done